### ISSUE TITLE
Allow opt in to show Apple Pay button

### ISSUE THREAD

**Issue by meismyles** [braintree/braintree-ios-drop-in#24]
Currently, you allow us to opt out of showing Apple Pay, even if it is supported by setting the following:

```swift
let request =  BTDropInRequest()
request.applePayDisabled = true // If you'd like to opt out
```

However, you don't seem to allow the reverse - the Apple Pay button remains hidden if the user has no cards available that support it, even if setting:
```swift
request.applePayDisabled = false
```

This is because in the following block of code in `BTPaymentCollectionViewController.m` - **line 220**:

```objc
#ifdef __BT_APPLE_PAY
            BTJSON *applePayConfiguration = self.configuration.json[@"applePay"];
            if ([applePayConfiguration[@"status"] isString] && ![[applePayConfiguration[@"status"] asString] isEqualToString:@"off"] && !self.dropInRequest.applePayDisabled) {
                // Short-circuits if BraintreeApplePay is not available at runtime
                if (__BT_AVAILABLE(@"BTApplePayClient") && [configuration canMakeApplePayPayments]) {
                    [activePaymentOptions addObject:@(BTUIKPaymentOptionTypeApplePay)];
                }
            }
#endif
```

the inner check `[configuration canMakeApplePayPayments]` will return false in the scenario I mention above.

It would be nice if we could force the Apple Pay button to show and then we can push the user to the 'Add Card' screen within the iOS Wallet.

This has 2 key benefits to users:
1. It direct them immediately to the Wallet to add a card for Apple Pay if this is available.
2. It displays to the user that our app supports Apple Pay.

Would be interested to hear any thoughts?

**Comment by demerino** [comment]
@meismyles Thanks for the feedback. We'll look into adding this feature - it does sound reasonable.

**Comment by demerino** [comment]
Fixed in `5.2.0`.
